varnish/assets/configuration/spec.yaml

```
@@ -0,0 +1,86 @@
+name: Varnish
+files:
+  - name: varnish.yaml
+    options:
+      - template: init_config
```

```suggestion
      - template: init_config
         options:
         - template: init_config/default
```
include the default options at inti_config level